RIAssigner/data/PandasData.py

```diff
@@ -28,8 +29,14 @@ def retention_times(self) -> Iterable[int]:
     def retention_indices(self):
         if self._carbon_number_index is not None:
             return self._data[self._carbon_number_index] * 10
+        elif "retention_index" in self._data.columns:
+            return self._data["retention_index"]
         raise KeyError("Dataset does not contain retention indices!")

```

Comment:
Maybe this could also be changed so that upon opening a file and reading the data, the "internal" retention index column is initialized with whatever is in the file, let it be RI values or carbon numbers which are then converted to RI